src/senselab/audio/tasks/text_to_speech/__init__.py

```diff
@@ -1 +1,3 @@
-"""This module provides the API for the senselab text to speech task."""
+""".. include:: ./doc.md"""  # noqa: D415
```

Comment:
If it becomes common for us to use the doc.md method then let's add D415 to the ignore list.

Reply:
I would recommend keeping track of how often this method is used and evaluating if it’s truly beneficial before making the change permanent